compiler/codegen/symbol.cpp

```diff
@@ -56,6 +54,10 @@
 #include <inttypes.h>
 #include <stdint.h>
 
+#ifdef HAVE_LLVM
+#include "llvm/IR/Attributes.h"
+#endif
```

Comment:
I don't like putting these ifdefs here, but I can't see the default place where to put these ifdefs.